static/app/views/settings/project/tempest/index.tsx

```diff
@@ -1,98 +1,69 @@
 
-import {CredentialRow} from './CredentialRow';
-import EmptyState from './EmptyState';
+import DevKitSettings, {getDevKitHeaderAction} from './DevKitSettings';
+import PlayStationSettings, {getPlayStationHeaderAction} from './PlayStationSettings';
 
 interface Props {
   organization: Organization;
   project: Project;
 }
 
+type Tab = 'playstation' | 'devkit-crashes';
+
+const TAB_LABELS: Record<Tab, string> = {
+  playstation: t('Retail'),
+  'devkit-crashes': t('DevKit'),
+};
+
+const PS5_WARNING_DISMISS_KEY = 'tempest-ps5-warning-dismissed';
+
 export default function TempestSettings({organization, project}: Props) {
   const hasWriteAccess = useHasTempestWriteAccess();
-  const {
-    data: tempestCredentials,
-    isLoading,
-    invalidateCredentialsCache,
-  } = useFetchTempestCredentials(organization, project);
-
-  const api = useApi();
-  const {mutate: handleRemoveCredential, isPending: isRemoving} = useMutation<
-    unknown,
-    RequestError,
-    {id: number}
-  >({
-    mutationFn: ({id}) =>
-      api.requestPromise(
-        `/projects/${organization.slug}/${project.slug}/tempest-credentials/${id}/`,
-        {
-          method: 'DELETE',
-        }
-      ),
-    onSuccess: () => {
-      addSuccessMessage(t('Removed the credentials.'));
-      trackAnalytics('tempest.credentials.removed', {
-        organization,
-        project_slug: project.slug,
-      });
-      invalidateCredentialsCache();
-    },
-    onError: error => {
-      const message = t('Failed to remove the credentials.');
-      handleXhrErrorResponse(message, error);
-      addErrorMessage(message);
-    },
-  });
-
-  const credentialErrors = useMemo(() => {
-    return tempestCredentials?.filter(
-      credential => credential.messageType === MessageType.ERROR && credential.message
-    );
-  }, [tempestCredentials]);
-
-  useEffect(() => {
-    if (credentialErrors && credentialErrors.length > 0) {
-      trackAnalytics('tempest.credentials.error_displayed', {
-        organization,
-        project_slug: project.slug,
-        error_count: credentialErrors.length,
-      });
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [credentialErrors]);
+  const location = useLocation();
+  const navigate = useNavigate();
+  const {dismiss: dismissPS5Warning, isDismissed: isPS5WarningDismissed} =
+    useDismissAlert({
+      key: PS5_WARNING_DISMISS_KEY,
+    });
 
-  const isEmpty = useMemo(() => {
-    return !tempestCredentials?.length;
-  }, [tempestCredentials]);
+  const getTabFromQuery = (): Tab => {
+    const queryTab = decodeScalar(location?.query?.tab);
+    return (
+      ['playstation', 'devkit-crashes'].includes(queryTab || '')
+        ? queryTab
+        : 'playstation'
+    ) as Tab;
+  };
+
+  const [tab, setTab] = useState<Tab>(getTabFromQuery());
+
+  const handleTabChange = (newTab: Tab) => {
+    setTab(newTab);
+
+    navigate({
```

Comment:
why do we need `tab` in local state if we also push `tab` to the url? Couldn’t we just read it from the url as the single source of truth?